test/pages/uat_actions/test_runs/_id.spec.js

```diff
@@ -0,0 +1,121 @@
+import TestRun from '@/pages/uat_actions/test_runs/_id'
+import { createLocalVue, mount } from '@vue/test-utils'
+import { BootstrapVue } from 'bootstrap-vue'
+import lighthouse from '@/modules/lighthouse_service'
+import sprint from '@/modules/sprint'
+
+jest.mock('@/modules/lighthouse_service')
+jest.mock('@/modules/sprint')
+
+const localVue = createLocalVue()
+localVue.use(BootstrapVue)
+
+describe('TestRuns.vue', () => {
+  let wrapper, testRunData, $route
+
+  beforeEach(() => {
+    $route = {
+      params: {
+        id: 1
+      }
+    }
+    testRunData = {
+      "id": 1,
+      "created_at": "2021-07-02T09:00:00.000Z",
+      "updated_at": "2021-07-12T11:31:15.806Z",
```

Comment:
I think the date fields are called `_created` and `_updated` and that the ID is `_id`, but I may be wrong?